src/black/__init__.py

```diff
@@ -653,7 +633,11 @@ def get_sources(
         if is_stdin or p.is_file():
             normalized_path = normalize_path_maybe_ignore(p, ctx.obj["root"], report)
```

Comment:
Seems like the mypyc build is broken here (see the diff-shades output in CI); it infers None for this variable.